cvat/scripts/pascal_voc.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
The script is very nice. Probably it will be very convenient for other users to have a short description what it does and an example how to run it.

Reply:
Should that description go above the imports or in the description argument of argparser?

Reply:
Above imports. Also what is the license for you script? MIT?